src/vesper-metadata.json

```diff
@@ -59,7 +66,13 @@
       "symbol": "veFRAX-FRAXBP",
       "decimals": 18,
       "logoURI": "https://raw.githubusercontent.com/vesperfi/metadata/master/src/logos/vefrax-fraxbp.svg",
-      "type": "earn"
+      "type": "earn",
+      "collateral": {
+        "address": "0x853d955aCEf822Db058eb8505911ED77F175b99e",
+        "decimals": "18",
```

Comment:
Should use `Number` for decimals everywhere.

Reply:
Agree. Fixed.